packages/v4/patternfly-docs/content/design-guidelines/components/menu/menu.md

```diff
@@ -97,6 +97,35 @@ When using red text for destructive actions, it is still recommended to require
 
 <img src="./img/red-delete-text.png" alt="drilldown menu" width="935"/>
 
+### Disabling menus and menu options
+Menus or menu items can be disabled for multiple reasons. The reason an action is disabled informs which design pattern you should use.
+
+1. **An action can be performed, but is not recommended.**
+When an action can be performed but may result in an undesirable outcome, do not disable it. 
+
+Instead, add a [confirmation model](components/model). When the user clicks on the action, use a modal to explain the potential consequences and ask the user if they are sure they want to proceed.
+
+Example: A user wants to start maintenance on a master node.
```

Comment:
Having this first item is confusing because you're talking about a case where something should NOT be disabled. I would probably make this last in the list, and also change the example to something more tangible like "Example: A user wants to delete a system."